service/src/test/java/com/commercetools/pspadapter/payone/IntegrationServiceTest.java

```diff
@@ -72,52 +63,20 @@ public void tearDown() throws Exception {
 
     @Test
     public void createHealthResponse_tenantConfigIsWorking_shouldReturnOkStatus() {
```

Comment:
We are actually not checking tenant configs anymore. So I'd remove it from the test name.

Reply:
https://github.com/commercetools/commercetools-payone-integration/pull/236/commits/f558a77fbeb34001167daddefd716ccc8224e0eb#diff-93bd210e428df4a1fb1d0a0c19798021